book/src/getting-started.md

```diff
@@ -117,6 +116,40 @@ demo completes after it has convinced itself the testnet won't process any addit
 transactions. You should see several TPS measurements printed to the screen. In the
 multinode variation, you'll see TPS measurements for each validator node as well.
 
+### Testnet Debugging
+
+There are some useful debug messages in the code, you can enable them on a per-module and per-level
+basis.  Before running a leader or validator set the normal RUST\_LOG environment variable.
+
+For example
+
+* To enable `info` everywhere and `debug` only in the solana::banking_stage module:
+
+  ```bash
+  $ export RUST_LOG=info,solana::banking_stage=debug
```

Comment:
How about this:
`export RUST_LOG=solana=info,solana::banking_stage=debug`
A unqualified `info` will cause `cargo` info logs to show up too and we don't usually care about those